gdcdictionary/schemas/treatment.yaml

```diff
@@ -113,54 +212,217 @@ properties:
       - Genitourinary, NOS
       - Genitourinary, Prostate
       - Genitourinary, Prostate and Seminal Vesicles
+      - Groin
+      - Gum
+      - Hand
+      - Hard Palate
       - Head
+      - Head and Neck
+      - Head - face or neck, NOS
       - Head, Face, or Neck
+      - Heart
+      - Hepatic
+      - Hepatic Duct
+      - Hepatic Vein
       - Hilar
+      - Hip
+      - Hippocampus
+      - Humerus
+      - Hypopharynx
+      - Ileocecal Valve
+      - Ileum
       - Iliac-common
       - Iliac-external
+      - Ilium
+      - Index finger
       - Inguinal
       - Internal Mammary Nodes
+      - Ischium
+      - Islet Cells
+      - Jaw
+      - Jejunum
+      - Joint
+      - Kidney
+      - Knee
+      - Lacrimal Gland
+      - Large Bowel
+      - Laryngopharynx
+      - Larynx
       - Leg
+      - Leptomeninges
+      - Ligament
+      - Lip
+      - Liver
       - Local Recurrence
+      - Lumbar Spine
       - Lung
+      - Lymph Node
+      - Lymph Node(s) Axilla
+      - Lymph Node(s) Cervical
+      - Lymph Node(s) Epitrochlear
+      - Lymph Node(s) Femoral
+      - Lymph Node(s) Hilar
+      - Lymph Node(s) Iliac-common
+      - Lymph Node(s) Iliac-external
+      - Lymph Node(s) Inguinal
+      - Lymph Node(s) Internal Mammary
+      - Lymph Node(s) Mammary
+      - Lymph Node(s) Mesenteric
+      - Lymph Node(s) Occipital
+      - Lymph Node(s) Parotid
+      - Lymph Node(s) Pelvic
+      - Lymph Node(s) Popliteal
+      - Lymph Node(s) Retroperitoneal
+      - Lymph Node(s) Splenic
+      - Lymph Node(s) Subclavicular
+      - Lymph Node(s) Submandibular
+      - Lymph Node(s) Supraclavicular
       - Lymph Nodes
+      - Lymph Nodes(s) Mediastinal
```

Comment:
It's from the ticket. I take it this is an intentional misspelling?

Reply:
Good catch! All of the rest of the "Lymph Node(s)" seem to be okay. Unless this will de-synchronize us from some external list, I think it should be corrected.